source/MaterialXGenShader/GenContext.h

```diff
@@ -126,7 +126,7 @@ class GenContext
 
     /// Find and return a cached shader node implementation,
     /// or return nullptr if no implementation is found.
-    ShaderNodeImplPtr findNodeImplementation(const string& name);
+    ShaderNodeImplPtr findNodeImplementation(const string& name) const;
```

Comment:
This change is not related to the fix but still help to understand the code more quickly (since it was the first time I was looking at this part of the MaterialX code).